docs/index.rst

```diff
@@ -908,6 +908,85 @@ The go ahead and add this new resource representation to your `FlaskView`::
             return {'This is JSON': 'How Cool is that'}
 
 
+The ``representations`` is a dictionary in which the key is the accepted content type and the value
+is a ``flask.make_response`` proxy function with the same signature.
+
+By default, the ``representations`` class attribute is an empty dictionary.
+
+When the ``representations`` is not empty and if the view functions returns the
+``flask.wrappers.ResponseBase`` instance, it will be returned immediately to the ``Flask`` to handle
+the rest. Otherwise, ``Flask-Classful`` will try to find the best match of the ``representations``'s
+accepted content type and call the associated output proxy function to create the
+``flask.wrappers.ResponseBase`` instance. If no matching output proxy function is found
+when ``Flask-Classfull`` looks up, it will call the first entry's value of the dictionary (This
+behavior could change in the future, maybe just return the data?).
```

Comment:
Couple things. First off, to handle a missing content type from `representations` we should either throw a specific exception or provide the usage of like a `default` key in the representations dictionary that is a catch-all function that can always be used. Alternatively, we should just return the data we were given. I don't like using the first key in the dictionary, because dictionary ordering isn't guaranteed until Py3.6, in earlier versions you'd get a random key every time and that sounds SUPER frustrating.

Second, I'm going to reword some of these paragraphs because some of the phrasing is a tad awkward. Here's a reword of the above:

```rst
When the ``representations`` dictionary is not empty and if the view function returns a
``flask.wrappers.ResponseBase`` instance, it will be returned immediately to ``Flask`` to handle
the rest. Otherwise, ``Flask-Classful`` will try to find the best match between the accepted content
type and the keys in the ``representations`` dictionary, and call the associated output proxy
function to create a ``flask.wrappers.ResponseBase`` instance. If no matching output proxy function
is found when ``Flask-Classful`` looks up, it will call the first key's value in the dictionary (This
behavior could change in the future, maybe just return the data?).
```

Also, can we define "find the best match"? For example, if I pass it `application/json` as my content type, will it look for `application/json`, `json`, and `application` in that order? Or just `application/json`?

Reply:
thank you for your valuable feedback, let's continue the discussion about the missing content type case and any related concern about representations with this issue https://github.com/teracyhq/flask-classful/issues/72